AUTHORS.rst

```diff
@@ -0,0 +1,10 @@
+Authors
+=======
+
+Billinge Group and community contributors.
```

Comment:
let's add your name and Andrew's name explicitly as well as this general statement.  I think you both were the genesis of this project.

Reply:
done 

```
Authors
=======

Sangjoon Lee (sl5400@columbia.edu)
Andrew Yang (ay2546@columbia.edu)
Simon Billinge (sb2896@columbia.edu)

Billinge Group and community contributors
```